Woof/Woof/View/OwnerFlow/OwnerOnboardingView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+/// A view that onboards the owner and also collects mandatory information about them.
+struct OwnerOnboardingView: View {
+    /// The name of the owner.
+    @Binding var name: String
+
+    /// The surname of the owner.
+    @Binding var surname: String
+
+    /// The phone of the owner.
+    @Binding var phone: String
+
+    /// The owner's city.
+    @Binding var city: String
+
+    /// The address of the owner street and building
+    @Binding var address: String
+
+    /// A boolean indicating whether the save button should be enabled.
+    var isSaveButtonEnabled: Bool {
+        !name.isEmpty && !phone.isEmpty && !city.isEmpty && !address.isEmpty
+    }
+
+    var body: some View {
+        VStack {
+            Text(welcomeText)
+                .foregroundColor(Color.App.purpleDark)
+
+            Text(onboardingText)
+                .lineLimit(lineLimitForOwnerOnboarding)
+                .foregroundColor(Color.App.purpleDark)
+
+            EditOwnerInformationView(
+                name: $name,
+                surname: $surname,
+                phone: $phone,
+                city: $city,
+                address: $address
+            )
+
+            Button(proceedButtonTitle) {}
+                .padding()
+                .clipShape(Capsule())
+                .disabled(!isSaveButtonEnabled)
+                .background(isSaveButtonEnabled ? Color.App.purpleDark : Color.gray)
+                .foregroundColor(Color.App.white)
```

Comment:
why don't use the existing button modifier?
```suggestion
           Button(proceedButtonTitle) {}
                .buttonStyle(PurpleCapsuleOfInfinityWidth())
                .disabled(!isSaveButtonEnabled)
```

Just need to update the style a little bit to support a disabled modifier. Check the implementation of `CapsuleWithWhiteText`

Reply:
Added with e8e2fd4